t/discovery/nacos.t

```diff
@@ -251,3 +251,134 @@ discovery:
 ]
 --- no_error_log
 [error]
+
+
+
+=== TEST 8: get APISIX-NACOS info from NACOS - no auth with namespace
```

Comment:
Need another test that the request gets 503 because the namespace is incorrect.

Reply:
fixed.